index.html

```diff
@@ -0,0 +1,1342 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+        integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css">
+    <title>News week clone</title>
+</head>
+
+<body class="container-xxl p-0">
+    <header class="container-fluid p-0">
+        <nav class="desktop-nav ">
+            <div class="first-header d-flex justify-content-between align-items-center">
+                <div class="d-none d-lg-block align-self-end text-white">
+                    Web, Jan 20, 2021
+                </div>
+                <div class="header-svg px-2 w-100">
+                    <svg version="1.1" width="396" height="60" xmlns="http://www.w3.org/2000/svg"
```

Comment:
- [ ] Consider saving your SVG images in an .image_name.svg file and then just calling it's path with the image tag in the HTML.
That will make your HTML file easier to read through and smaller, plus your repo more organized!